### DISCUSSION TITLE
Firebase support?

### DISCUSSION THREAD

**Discussion by Flajt** [deta/cloud-docs#622] [Help] [answered]
I'm currently building a very small application which depends on firebase to authenticate mobile devices.
If I try to access my route it fails with a 502 error, so I assume that something with firebase doesn't work, (logs don't show any error, neither the visor nor the cli).

The docs mention that firebase seems not to install correctly. So I wanted to inquire if there is something that can be done, or if I need to migrate to a different provider for now.

**Comment by jnsougata** [comment, marked as answer]
It's a known issue. Libraries from Google don't usually work on micros

**Reply by Flajt** [reply]
Yeah this is what I understood aswell. 
I guess I will move stuff somewhere else. Ty for the reply

**Comment by jhihyulin** [comment]
If you're talking about Firebase_admin, I also have the same error, but changed to version 5.0.2 it works fine.

**Reply by Flajt** [reply]
I'm using version 11.0.1, I think that would be to much of a downgrade.

**Reply by jhihyulin** [reply]
Well, I seems that you are using Node.js, because the latest version for Python is v6.0.1.

**Reply by Flajt** [reply]
Ah yes, sry I'm using node.

Interesting, that it works with python but not node. Maybe the issue is somewhere else.

**Comment by gamemaker1** [comment]
Hi,

I'm also using Node 14 + Fastify and trying to deploy on Deta, however, any request to the endpoint returns a 502 Bad Gateway error almost instantly. I'm not using any Google libraries, although, I do call the Firebase REST API a few times in my code.

Could anyone point out what might be going wrong?

Thanks!
